### PR TITLE
remove schema if required

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: zhuxiaoyang <sunzhu@yunify.com>

### CODE REVIEW

**Comment by soulseen** [comment]
fix https://kubesphere.com.cn/forum/d/158-source-to-image

File: pkg/api/types.go [hunk 1]
```diff
@@ -747,6 +747,12 @@ func Parse(originalName, serverAddress string) (ref string, err error) {
 		return "", fmt.Errorf("parsing image %q failed: %v", originalName, err)
 	}
 
+	// remove schema if required
+	if strings.HasPrefix(serverAddress, "http") {
```
Comment (runzexia) [inline pkg/api/types.go]: httpd:latest
http/xxx:latest

Comment (soulseen) [reply pkg/api/types.go]: updated

**Pull request merged**